database/scripts/seed.ts

```diff
@@ -0,0 +1,191 @@
+            questionNumber: (i % QUESTIONS_PER_GROUP) + 1,
+        });
+    }
+    return questions;
+}
+
+function mockGroups() {
+    const groups = [];
+
+    for (let i = 0; i < GROUP_COUNT; i++) {
+        const questionIDs = [];
+
+        for (let j = 0; j < QUESTIONS_PER_GROUP; j++) {
+            questionIDs.push(i * QUESTIONS_PER_GROUP + j);
+        }
+        groups.push({ questionIDs });
+    }
+    return groups;
+}
+
+function mockComponents() {
+    const componentTypes = ["text", "video", "audio"];
+    const components = [];
+
+    for (let i = 0; i < COMPONENT_COUNT; i++) {
+        components.push({
+            _id: i,
+            type: componentTypes[i % componentTypes.length],
+            properties: {},
+        });
+    }
+    return components;
+}
+
+function mockSlides() {
+    const slides = [];
+
+    for (let i = 0; i < SLIDE_COUNT; i++) {
+        const componentIDs = [];
+
+        for (let j = 0; j < COMPONENTS_PER_SLIDE; j++) {
+            componentIDs.push(i * COMPONENTS_PER_SLIDE + j);
+        }
+
+        slides.push({
+            _id: i,
+            componentIDs,
+            prevID: i > 0 ? i - 1 : null,
+            nextID: i < SLIDE_COUNT - 1 ? i + 1 : null,
+        });
+    }
+    return slides;
+}
+
+function mockModules() {
+    const statusTypes = ["complete", "published", "draft"];
+    const modules = [];
+
+    for (let i = 0; i < MODULE_COUNT; i++) {
+        const slideIDs = [];
+
+        for (let j = 0; j < SLIDES_PER_MODULE; j++) {
+            slideIDs.push(i * SLIDES_PER_MODULE + j);
+        }
+
+        modules.push({
+            _id: i,
+            title: faker.lorem.words(),
+            toolID: i < TOOL_COUNT ? i : null,
+            slideIDs,
+            status: statusTypes[i % statusTypes.length],
+            editing: i % statusTypes.length == 2,
+        });
+    }
+    return modules;
+}
+
+function mockTools() {
+    const statusTypes = ["published", "draft"];
+    const modules = [];
+
+    for (let i = 0; i < TOOL_COUNT; i++) {
+        const slideIDs = [];
+
+        for (let j = 0; j < SLIDES_PER_MODULE; j++) {
+            slideIDs.push(i * SLIDES_PER_MODULE + j);
+        }
+
+        modules.push({
+            _id: i,
+            title: faker.lorem.words(),
+            video: faker.internet.url(),
+            description: faker.lorem.sentences(),
+            moduleID: i < MODULE_COUNT ? i : null,
+            resources: [
+                {
+                    title: faker.lorem.words(),
+                    description: faker.lorem.sentences(),
+                    url: faker.internet.url(),
+                },
+                {
+                    title: faker.lorem.words(),
+                    description: faker.lorem.sentences(),
+                    url: faker.internet.url(),
+                },
+            ],
+            selfCheckGroupID: i < GROUP_COUNT ? i : null,
+            relatedToolsIDs: i > 0 ? [0] : null,
```

Comment:
do we want to randomly pick some related tools just so we have some data for this?

Reply:
I randomized the `relatedToolsIDs` of each tool and left some comments